tests/checks/integration/test_process.py

```diff
@@ -269,13 +269,13 @@ def mock_get_pagefault_stats(pid):
             'instances': [stub['config'] for stub in self.CONFIG_STUBS]
         }
 
-        self.run_check(config, mocks=mocks)
+        self.run_check_twice(config, mocks=mocks)
 
         instance_config = config['instances'][-1]
         for stat_name in self.PAGEFAULT_STAT:
-            self.assertMetric('system.processes.mem.page_faults.' + stat_name, at_least=1,
+            self.assertMetric('system.processes.mem.page_faults.' + stat_name,
                 tags=self.generate_expected_tags(instance_config),
-                value=pg_fault_stats[stat_name])
```

Comment:
Ah damn. I just ran the test with `nosetests tests/checks/integration/test_process.py:ProcessCheckTest.test_check`.

I'll fix it. Thank you!